package.json

```diff
@@ -139,9 +141,9 @@
             "csharp"
           ]
         },
-        "runtime": "node",
+        "runtime": "",
         "runtimeArgs": [],
-        "program": "./out/coreclr-debug/proxy.js",
+        "program": "./coreclr-debug/debugAdapters/OpenDebugAD7.exe",
         "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
         "configurationAttributes": {
           "launch": {
```

Comment:
You need to undo these changes in the debuggers section. This change occurs because the extension rewrites this portion of its manifest during activation. F5'ing the extension in dev scenarios causes this rewrite to occur, but the changes cannot be checked in.